src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Computer desktop"
+        class="card__image"
+      >
+
+      <p class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="card__subtitle">
+        Код товара: 195434
+      </p>
+
+      <div class=" card__rating">
+        <div class="stars">
+          <span class="stars__star stars__star--4"></span>
+          <span class="stars__star stars__star--4"></span>
+          <span class="stars__star stars__star--4"></span>
+          <span class="stars__star stars__star--4"></span>
+          <span class="stars__star "></span>
+        </div>
```

Comment:
don't hardcode active stars. Add modifier to the whole block stars (`stars stars--4`) and write styles for all the possible modifiers, the way you did in layout_stars task.